includes/EditPage.php

```diff
@@ -1878,14 +1878,14 @@ function showEditForm( $formCallback = null ) {
 		}
 
 		// wikia change begin
-		wfRunHooks ('EditForm:BeforeDisplayingTextbox', array (&$this, &$hidden /* TODO: remove? */) ) ;
+		wfRunHooks ('EditForm:BeforeDisplayingTextbox', array ($this, &$hidden /* TODO: remove? */) ) ;
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)